frontend/src/routing/associations.ts

```diff
@@ -9,13 +9,35 @@ import { AssociationFilesystemDetail } from "../pages/associations/medias/Detail
 import { AssociationFilesystemEdit } from "../pages/associations/medias/Edit";
 import { AssociationFilesystemUpload } from "../pages/associations/medias/Upload";
 
+// To be replaced
+import { AssociationEventList } from "../pages/associations/events/List";
+import { AssociationEventCreate, AssociationEventEdit } from "../pages/associations/events/Edit";
+
 export const routes = association => [
     {
         path: `/`,
         component: AssociationHome,
         exact: true,
         props: { association: association }
     },
+    {
+        path: `/events/`,
+        component: AssociationEventList,
+        exact: true,
+        props: { association: association }
+    },
+    {
+        path: `/events/new`,
```

Comment:
Please use French for the URLs.
=> `/evenements/nouveau/`, etc.

Reply:
I don't think I get it... All the other URLs and the backend have english URLs... Why would the events be of a difference ?

Reply:
The backend is internal, so the URLs should be kept in English
However, the URLs are seen by the end-users, they are part of the public interface. They thus should be in French.
The other URLs should be changed too, but we'll deal with it later

Reply:
This should be done in a separate branch, since all the URLs are yet in english, it would bring more inconsistency in my opinion....